graphql-server/Tests/AppTests/AppTests.swift

```diff
@@ -0,0 +1,42 @@
+import App
+import Hummingbird
+import HummingbirdXCT
+import XCTest
+
+final class AppTests: XCTestCase {
+    static let allTests = [
+        ("testGraphQLSuccess", testGraphQLSuccess),
+    ]
```

Comment:
Don't tend to include `allTests` anymore. Depend on --enable-test-discovery, also swift 5.4 doesn't need them. Given your `LinuxMain.swift` is empty there is no reason for this at all.

Reply:
Ah! I should be on top of these things. TIL. Thanks! Done.